src/sync/queries.ts

```diff
@@ -52,6 +52,7 @@ export const getCommits = `query ($owner: String!, $repo: String!, $per_page: In
                   message
                   oid
                   url
+                  changedFiles
```

Comment:
This is a number and not a list of changed files, right? Just want to prevent us from loading a bunch of data that we end up not needing.

Reply:
Correct, just an `Int`.  We weren't actually sending the commited files to the jira api, just 0.